lib/client/vm_api.rb

```diff
@@ -6,7 +6,11 @@ def vm(vm_id, opts={})
     end
 
     def vms(opts={})
-      headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      if opts[:without_details]
+        headers = {:accept => "application/xml"}
+      else
+        headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      end
       path = "/vms"
       path += search_url(opts) unless filtered_api
```

Comment:
The API supports two modes.
Filtered and unfiltered.
The filtered API is what we call a "user-level" API (rather than admin API).

I think that it is less relevant for Foreman, as you should only work with admin users, right?
Do you ever set the filtered_api flag?